app/controllers/chargeback_controller.rb

```diff
@@ -765,14 +769,18 @@ def get_cis_all
     unless WHITELIST_INSTANCE_TYPE.include?(klass)
       raise ArgumentError, "Received: #{klass}, expected one of #{WHITELIST_INSTANCE_TYPE}"
     end
-    classtype =
+    all_of_classtype =
       if klass == "enterprise"
-        MiqEnterprise
+        MiqEnterprise.all
+      elsif klass == "ext_management_system"
+        ExtManagementSystem.all.reject { |prov| prov.is_a? ManageIQ::Providers::ContainerManager }
```

Comment:
Yes, kind_of? is preferred, but no need to change unless we need other changes in this PR.